.github/workflows/apps-deploy.yml

```
@@ -34,16 +34,16 @@ jobs:
             type: "shinyappsio"
             account: "testing-apps"
             server_name: "shinyapps.io"
-            cores: 5
+            cores: 1
```

Shinyapps.io takes too long. Reverting

```suggestion
            cores: 5
```